docs/user-guide/managing-deployments.md

```diff
@@ -275,6 +293,22 @@ my-nginx-divi2   1/1       Running   0          1h
 my-nginx-o0ef1   1/1       Running   0          1h
 ```
 
+To have the system automatically choose the number of nginx replicas as needed, range from 1 to 3, do:
+
+```console 
+$ kubectl autoscale rc my-nginx --min=1 --max=3
+replicationcontroller "my-nginx" autoscaled
+$ kubectl get pods -lapp=nginx
+NAME             READY     STATUS    RESTARTS   AGE
+my-nginx-1jgkf   1/1       Running   0          3m
+my-nginx-divi2   1/1       Running   0          3m
+$ kubectl get horizontalpodautoscaler 
+NAME      REFERENCE                           TARGET    CURRENT     MINPODS   MAXPODS   AGE
+nginx     ReplicationController/nginx/scale   80%       <waiting>   1         3         1m
+```
+
+For more information, please see [kubectl scale](kubectl/kubectl_scale.md), [kubectl autoscale](kubectl/kubectl_autoscale.md) and [horizontal pod autoscaler](horizontal-pod-autoscaling/README.md) document.
+
 ## Updating your application without a service outage
 
 At some point, you’ll eventually need to update your deployed application, typically by specifying a new image or image tag, as in the canary deployment scenario above. `kubectl` supports several update operations, each of which is applicable to different scenarios.
```

Comment:
I agree, @kargakis. I do hope to steer most users to Deployment, but we might also want a rolling-update interface to Deployment.
cc @nikhiljindal